tests/output/ConsoleInteractionTest.py

```diff
@@ -634,7 +634,8 @@ def test_print_section_beginning(self):
 
     def test_nothing_done(self):
```

Comment:
Since the function doesn't exist anymore as well as the parameter. So, IMO this function is of no need now. Also, in issue description its mentioned that we don't need this test